client/src/components/DisplayBookings/BookingGroup/BookingGroup.tsx

```diff
@@ -0,0 +1,54 @@
+import { Grid, Paper, Box, Typography } from '@material-ui/core';
+import { BookingApiData } from '../../../interface/BookingApiData';
+import SelectBooking from '../SelectBooking/SelectBooking';
+import BookingCard from '../BookingCard/BookingCard';
+import useStyles from './useStyle';
+
+interface Props {
+  bookings: BookingApiData;
+}
+
+export default function BookingGroup(props: Props): JSX.Element {
+  const classes = useStyles();
+  const { bookings } = props;
+  const { upcoming, current, past } = bookings;
+  return (
+    <Grid item lg={4} md={5} sm={6} xs={11} container justify="center">
+      <Box className={classes.bookingGroupContainer}>
+        <Grid container direction="column" justify="space-evenly" alignItems="center">
+          <Grid component={Paper} elevation={5} item className={classes.upcoming}>
+            <Box>
+              <Box display="flex" justifyContent="space-between">
+                <Typography variant="h6" className={classes.label}>
+                  YOUR NEXT BOOKING:
```

Comment:
Please avoid using CAPITALCASE text as some screen readers interpret them as abbreviations, use a CSS class to transform text instead.